tests/ignite/contrib/conftest.py

```diff
@@ -1,14 +1,13 @@
 import random
-
 import pytest
 
 
 @pytest.fixture()
 def visdom_offline_logfile(dirname):
```

Comment:
We could also have `dirname` as Path object here instead of str. Dirname is a fixture defined in tests/ignite/conftest.py